### PR TITLE
nl translation of the full Ordinals Manual

### PR DESCRIPTION
**Pull request by Tibebtc**
nl translation of the full Ordinals Manual

### CODE REVIEW

**Comment by Tibebtc** [comment]
@raphjaph Did I submit the PO file in the correct way?

File: nl.po [hunk 1]
```diff
@@ -0,0 +1,6527 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: Handleiding voor de Ordinale Theorie\n"
+"POT-Creation-Date: 2024-06-29T00:05:48-05:00\n"
+"PO-Revision-Date: 2024-08-21 14:50+0200\n"
+"Last-Translator: Zerone @0xZerone <zerone495@gmail.com>\n"
```
Comment (raphjaph) [inline nl.po]: Is this your email? 

If not either update or get rid of this line

File: docs/po/nl.po [hunk 2]
```diff
@@ -0,0 +1,6527 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: Handleiding voor de Ordinale Theorie\n"
+"POT-Creation-Date: 2024-06-29T00:05:48-05:00\n"
```
Comment (raphjaph) [inline docs/po/nl.po:4]: Maybe update these or delete if you don't want your email here

**Review by raphjaph — CHANGES_REQUESTED** [review]

**Review by raphjaph — APPROVED** [review]
LGTM

**Pull request merged**